src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -1052,7 +1056,11 @@ def __call__(
                         callback(i, t, latents)
 
         if not output_type == "latent":
-            image = self.vae.decode(latents / self.vae.config.scaling_factor, return_dict=False)[0]
+            condition_kwargs = {}
+            if isinstance(self.vae, AsymmetricAutoencoderKL):
+                mask_condition = mask_condition.to(device=device, dtype=masked_image_latents.dtype)
```

Comment:
Can we maybe compute the init_image_condition only here? Since it's not needed for the "normal" VAE?